libs/transactions-journey/src/lib/transactions-journey-shell.config.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+
+export type TransactionsJourneyExtensionsConfig = Record<string, unknown>;
+export const TRANSACTION_EXTENSIONS_CONFIG =
```

Comment:
Looks like this is a duplicate of the existing token [here](https://github.com/Backbase/golden-sample-app/blob/a7653afcb6d0fe4958f31a58915543a474d053fa/libs/transactions-journey/internal/feature-transaction-view/src/lib/extensions/config.ts#L9) - this is probably why you're having to provide it twice in the journey bundle, once for each token.